kalite/topic_tools/__init__.py

```diff
@@ -46,11 +46,22 @@
     logging.warning("Channel {channel} does not exist.".format(channel=django_settings.CHANNEL))
 
 
+def cache_file_path(basename):
+    """Consistently return path for a cache filename. This path has to be
+    writable for the user running kalite."""
+    assert "/" not in basename, "Please use a valid filename"
```

Comment:
Yeah, mainly that the channel name was being used for one of the file names and I didn't know where the channel names were coming from... very much not important assertion.